qa/tasks/ses_qa.py

```diff
@@ -178,6 +178,142 @@ def _get_drive_group_limit(self, **kwargs):
                            if 'deepsea' in x and 'drive_group' in x['deepsea'])
         return int(drive_group['deepsea']['drive_group']['custom']['data_devices']['limit'])
 
+    def ses_rack_dc_region_unavailability(self, **kwargs):
+        """
+        Simulates rack, DC and region unavailability by 
+        modifying Ceph crushmap
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_rack_dc_region_unavailability.sh',
+                )
+    
+    def ses_network_failure(self, **kwargs):
+        """
+        Simulates network failure using tc netem tool
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_network_failure.sh',
+                )
+
+    def ses_cephfs_test_mount(self, **kwargs):
+        self.scripts.run(
+                self.master_remote,
+                'ses_cephfs_test_mount.sh',
```

Comment:
insert here:

```
            args=mon_hosts
```

Reply:
getting error:

`    
args=mon_hosts,
NameError: global name 'mon_hosts' is not defined
`

Reply:
you missed https://github.com/smithfarm/ceph/pull/8/files/d10988fdbf0200f473120c828146d4eac8181eba#r343038202 ?